src/metaswitch/clearwater/cluster_manager/plugin_utils.py

```diff
@@ -37,6 +38,31 @@ def write_cluster_settings(filename, cluster_view):
         f.write(new_file_contents)
 
 
+def write_chronos_cluster_settings(filename, cluster_view, current_server):
+    current_or_joining_servers = [constants.JOINING_ACKNOWLEDGED_CHANGE,
+                                  constants.JOINING_CONFIG_CHANGED,
+                                  constants.NORMAL_ACKNOWLEDGED_CHANGE,
+                                  constants.NORMAL_CONFIG_CHANGED,
+                                  constants.NORMAL]
+    leaving_servers = [constants.LEAVING_ACKNOWLEDGED_CHANGE,
```

Comment:
It's a bit odd that `leaving_servers` contains some states and `leaving` includes the servers that are leaving - better variable names?